libs/wingsdk/src/target-sim/bucket.inflight.ts

```diff
@@ -300,6 +347,6 @@ export class Bucket implements IBucketClient, ISimulatorResourceInstance {
   }
 
   private hashKey(key: string): string {
-    return crypto.createHash("sha512").update(key).digest("hex");
+    return crypto.createHash("sha512").update(key).digest("hex").slice(-16);
```

Comment:
Why?

Reply:
Just an optimization for error readability. I don't think there's a super high chance of files colliding in simulator (besides there are also limits to how long file names can be on different operating systems) - we can always tweak later if needed